boat_nav/src/path_planners/planner_base.py

```diff
@@ -181,29 +183,16 @@ def publish_target(*argv):
 	@staticmethod
 	def _calc_wind_coming():
```

Comment:
_ before calc?

Reply:
Private func. That said, might deserve refactoring. Let's continue in https://github.com/uwsailbot/boat_2018/pull/21#discussion_r236398697